authors/apps/authentication/models.py

```diff
@@ -74,6 +80,7 @@ class User(AbstractBaseUser, PermissionsMixin):
     # log into the Django admin site. For most users, this flag will always be
     # falsed.
     is_staff = models.BooleanField(default=False)
+    is_confirmed = models.BooleanField(default=False)
```

Comment:
add a comment for this so that everyone understands why its there

Reply:
Noted